home/processors.py

```diff
@@ -13,3 +13,9 @@ def show_footers(request):
     if start_link:
         show_footers_ = False
     return {'show_footers': show_footers_}
+
+
+def add_vapid_public_key(request):
```

Comment:
would it be better to keep this in the messaging module?

Reply:
done